drivers/dose/dose.c

```diff
@@ -187,9 +185,8 @@ static dose_signal_t state_transit_blocked(dose_t *ctx, dose_signal_t signal)
     (void) signal;
     uint32_t backoff;
 
-    backoff = random_uint32_range(xtimer_usec_from_ticks(xtimer_min_timeout),
-                                  2 * ctx->timeout_base);
-    xtimer_set(&ctx->timeout, backoff);
+    backoff = random_uint32_range(0, 2 * ctx->timeout_base);
```

Comment:
Just to be sure: What happens when setting a timeout that is below what the `periph_timer` implementation can achieve? Will it just use the smallest timeout instead or can the situation arise that no alarm is generated?

Reply:
For `periph_timer` there is a `min` field in ztimer `ztimer_periph->min`, so if for example calling `ztimer_set(ZTIMER_USEC, 0)`, then the callback will be actually set to `now + ztimer_periph->min`, which should guarantee the callback to trigger. Currently the default `CONFIG_ZTIMER_USEC_MIN` is 10, but this value can be tuned.